app-backend/db/src/test/scala/com/azavea/rf/database/PropTestHelpers.scala

```diff
@@ -116,4 +116,29 @@ trait PropTestHelpers {
   def fixupAoi(user: User, org: Organization, aoi: AOI): AOI = {
     aoi.copy(organizationId = org.id, owner = user.id, createdBy = user.id, modifiedBy = user.id)
   }
+
+  def fixupWorkspace(workspaceCreate: Workspace.Create, org: Organization, user: User): ConnectionIO[Workspace] = {
```

Comment:
In general, these `fixup` methods have been non-effectful -- do you think it's better to include the effectful part in the helpers? I'm open to the idea that it might be but otherwise I think we should keep it consistent